OpenRA.Mods.Common/Traits/Render/WithSpriteBody.cs

```diff
@@ -63,6 +66,16 @@ public string NormalizeSequence(Actor self, string sequence)
 			return RenderSprites.NormalizeSequence(DefaultAnimation, self.GetDamageState(), sequence);
 		}
 
+		// TODO: Get rid of INotifyBuildComplete in favor of using the upgrade system
+		public virtual void BuildingComplete(Actor self)
+		{
+			DefaultAnimation.PlayRepeating(NormalizeSequence(self, Info.Sequence));
+
+			if (Info.PauseAnimationWhenDisabled)
```

Comment:
Please move this `if` (and its contents) to the constructor so only the `PlayRepeating()` above remains in this method.